app/backend/approaches/chatreadretrieveread.py

```diff
@@ -335,12 +335,12 @@ def get_messages_from_history(
         message_builder = MessageBuilder(system_prompt, model_id)
 
         # Add examples to show the chat what responses we want. It will try to mimic any responses and make sure they match the rules laid out in the system message.
-        for shot in few_shots:
-            message_builder.append_message(shot.get("role"), shot.get("content"))
+        for shot in reversed(few_shots):
```

Comment:
nitpick: do we need to reverse order for few shots ? Does it make any difference ?

Reply:
Yep, that was actually the main point of this PR. The test will break without reversed() as they're getting put in wrong order now.